lib/icedfrisby.js

```diff
@@ -1,37 +1,34 @@
-/*jslint node: true */
-
-//
-// IcedFrisby.js
-// 2015 Robert Herhold (maintainer) & other wonderful contrubuters
-// 2011-2014 Vance Lucas, Brightbit, LLC
-//
-// IcedFrisby is a library designed to easily test REST API endpoints and their responses with node.js and Mocha.
-// It is based on the original Frisby project.
-//
-// IcedFrisby is distributed under the BSD license
-// http://www.opensource.org/licenses/bsd-license.php
-
-var nodeUtil = require('util');
-var pm = require('./pathMatch');
-var qs = require('qs');
-var util = require('util');
-var https = require('https');
-var request = require('request');
-var _ = require('lodash');
-var stackTrace = require('stack-trace');
-var fs = require('fs');
-var fspath = require('path');
-var Stream = require('stream').Stream;
-var chalk = require('chalk');
-var pkg = require('../package.json');
-
-// setup Chai
-var chai = require('chai');
-chai.config.includeStack = false;
-global.expect = chai.expect;
-
-// define the global setup defaults
-var getGlobalDefaults = _.constant({
+'use strict';
+
+
+/*
+ * IcedFrisby.js
+ * 2015 Robert Herhold (maintainer) & other wonderful contrubuters
+ * 2011-2014 Vance Lucas, Brightbit, LLC
+ *
+ * IcedFrisby is a library designed to easily test REST API endpoints and their
+ * responses with node.js and Mocha. It is based on the original Frisby project.
+ *
+ * IcedFrisby is distributed under the MIT and BSD licenses
+ * https://opensource.org/licenses/MIT
+ * https://opensource.org/licenses/bsd-license
+ */
+
+
+const _ = require('lodash');
+const chai = require('chai');
+const chalk = require('chalk');
+const https = require('https');
+const pkg = require('../package.json');
+const pm = require('./pathMatch');
+const qs = require('qs');
+const request = require('request');
+const stream = require('stream').Stream;
```

Comment:
The capital S in `Stream` signifies that it's a constructor function. It's a Crockford convention, and matches the module's convention. It makes more sense to keep the identifier capitalized.